video.md

```diff
@@ -45,8 +45,8 @@ The server sends a 16-byte header for each frame.
 | Offset | Size    | Value        |
 | ------ | ------- | ------------ |
 | 0      | 4 bytes | Payload size |
-| 4      | 4 bytes | 0            |
-| 8      | 4 bytes | 1            |
+| 4      | 4 bytes | itrack (0)   |
```

Comment:
What does `itrack` mean?

Reply:
Well good question, I also don't know.

This name comes from the [Bambu_Sample struct definition](https://github.com/bambulab/BambuStudio/blob/a4b3f7a7c73bafa7ccd5fdef3776184bc7bc9fea/src/slic3r/GUI/Printer/BambuTunnel.h#L97), I checked that it matched that field by modifying the frame header as mentioned previously.

Now this field is always unused by the Bambu Studio app so we can only speculate what it is...